apps/native-component-list/src/screens/Image/ImageSourcesScreen.tsx

```diff
@@ -0,0 +1,64 @@
+import { ImageContentFit, ImageContentPosition, Image } from 'expo-image';
+import React from 'react';
+import {
+  Dimensions,
+  Image as RNImage,
+  ImageResizeMode,
+  ScrollView,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+
+import { FunctionParameter, useArguments } from '../../components/FunctionDemo';
+import Configurator from '../../components/FunctionDemo/Configurator';
+import { Colors } from '../../constants';
+import { ResizableView } from './ImageResizableScreen';
+
+
+const parameters: FunctionParameter[] = [
+  {
+    name: 'Use React Native Image',
+    type: 'boolean',
+    initial: false,
+  },
+];
+
+
+
+const sources = [200, 400, 600, 800, 1000].map((size) => ({
+  uri: `https://picsum.photos/seed/${size}/${size}`,
+  width: size,
+  height: size
+}));
+
+export default function ImageSourcesScreen() {
```

Comment:
I've added an option to test multiple sources to the resizable screen in that PR: https://github.com/expo/expo/pull/20344. But maybe it would be better to have it on a separate screen. What do you think?